Docs makefile doesn't build

Repro:
```
cd docs
pip install -r requirements.txt
make html
```
fails with:
```
Running Sphinx v2.2.0

Extension error:
Could not import extension sphinxcontrib.googleanalytics (exception: cannot import name 'ExtensionError' from 'sphinx.application'
make: *** [html] Error 2
```

But my real question is, should it work?

--- a/docs/requirements.txt
+++ b/docs/requirements.txt
@@ -1,3 +1,3 @@
-sphinx
+sphinx==1.7.3
 sphinxcontrib-googleanalytics
 -e git://github.com/snide/sphinx_rtd_theme.git#egg=sphinx_rtd_theme

